src/test/resources/features/chatters/EditAndDelete.feature

```diff
@@ -0,0 +1,15 @@
+@chatter
+Feature: Edit Post
+
+  Background: Create Post
+    Given I go to "Chatter" Home Page
+    When I create a new Post with "Hi my name is Simon"
+
+  @deletePost
+  Scenario: Edit an existing Post
+    When I Edit the Post with "Hi my name is Simon Ovidio Miranda Chiri"
+    Then On the Chatter page should be display the "Hi my name is Simon Ovidio Miranda Chiri"
```

Comment:
This step could be more generic.
Then On the Chatter page should be displayed.
In the implementation use the helper like:
`assertTrue(postForm.isPostDisplayed(helper.getPostMessage()));`